examples/campaign_management/graduate_campaign_experiment.rb

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This example illustrates how to graduate a campaign experiment.
+
+require 'optparse'
+require 'google/ads/google_ads'
+require 'date'
+
+def graduate_campaign_experiment(customer_id, experiment_resource_name)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  # Graduating a campaign experiment requires a new budget. Since the budget
+  # for the base campaign has explicitly_shared set to false, it cannot be
+  # shared with the campaign after it is made independent by graduation.
+  budget_operation = client.operation.create_resource.campaign_budget do |b|
+    b.name = "Budget ##{(Time.new.to_f * 1000).to_i}"
+    b.amount_micros = 50_000_000
+    b.delivery_method = :STANDARD
+  end
+
+  response = client.service.campaign_budget.mutate_campaign_budgets(
+    customer_id,
+    [budget_operation]
+  )
+  budget_resource_name = response.results.first.resource_name
+
+  puts "Created new budget #{budget_resource_name} to add to experiment during graudation."
+
+  response = client.service.campaign_experiment.graduate_campaign_experiment(
```

Comment:
Since there are two steps here, it would be helpful if you had a one-line comment above line 46 that said "Graduates the campaign, using the budget created above" or something similar